rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
`ResourceType` is caused by `RxSwipeDismissBehaviorTestActivity#17` and `PrivateResource` is caused by referencing to `R.style.Theme_AppCompat`.